cleanlab/classification.py

```diff
@@ -416,6 +417,17 @@ class 0, 1, ..., K-1. `pred_probs` should have been computed using 3 (or
             not just the subset of cleaned data left after pruning out the label issues.
         """
 
+        if y is not None and labels is not None:
+            raise TypeError(
+                "CleanLearning.fit() received both `y` and `labels` arguments. "
+                "`labels` is deprecated, so please use `y` instead."
+            )
+        elif labels is not None:
+            warnings.warn("labels is deprecated; use y", DeprecationWarning, 2)
```

Comment:
Update message to read:

"Argument `labels` is deprecated and will be removed in future versions, pass the labels via argument `y` instead."